src/System.Net.Http/tests/FunctionalTests/ManagedHandlerTest.cs

```diff
@@ -391,5 +391,96 @@ public sealed class ManagedHandler_HttpClientHandler_ConnectionPooling_Test : Ht
                 }
             }
         }
+
+        [OuterLoop]
+        [Fact]
+        public async Task HttpConnectMethod_Succeeds()
+        {
+            using (HttpClient client = CreateHttpClient())
+            using (var listener = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp))
+            {
+                listener.Bind(new IPEndPoint(IPAddress.Loopback, 0));
+                listener.Listen(100);
+                var ep = (IPEndPoint)listener.LocalEndPoint;
+                var uri = new Uri($"http://{ep.Address}:{ep.Port}/");
+                string responseBody = "HTTP/1.1 200 Connection established\r\n\r\n";
```

Comment:
It's slightly weird that you're using "Connection established" as the reason phrase here, any reason for that?

Reply:
This is what my proxy server sends me (Squid on Linux). I can change that to what ever you think would be better. For now, I tried to mimic what I see in my private manual setup.

Reply:
Hmmm, ok.  It's not important, just struck me as weird.